app/main/views/brief_responses.py

```diff
@@ -230,21 +231,22 @@ def list_brief_responses():
         )
 
     if brief_id or supplier_id:
-        return jsonify(
-            briefResponses=[brief_response.serialize() for brief_response in brief_responses.all()],
+        brief_responses = brief_responses.all()
+        return list_result_response(
+            total_count=len(brief_responses),
+            briefResponses=[brief_response.serialize() for brief_response in brief_responses],
             links={'self': url_for('.list_brief_responses', supplier_id=supplier_id, brief_id=brief_id)}
```

Comment:
Will likely remove this individual link as I highly doubt it is being used. Either we can be consistent that all endpoints contain a link to themselves or none should.

Reply:
I think i's generally nice to include `self` link on our responses, but as we're the only consumer it doesn't matter too much.

Reply:
+1 to having a self link everywhere.
http://blog.ploeh.dk/2013/05/03/rest-lesson-learned-consider-a-self-link-on-all-resources/

Reply:
Interesting read. We could easily add this to all our resources using this PR. Not that it would probably have any immediate effects for us but there are things that we could do to produce more canonical URLs e.g. ordering query params.